programs/config/src/date_instruction.rs

```diff
@@ -21,15 +21,13 @@ pub struct DateConfig {
 impl Default for DateConfig {
     fn default() -> Self {
         Self {
-            date_time: Utc.timestamp(0, 0),
+            date_time: Utc.timestamp_opt(0, 0).unwrap(),
         }
     }
 }
 impl DateConfig {
-    pub fn new(date: Date<Utc>) -> Self {
```

Comment:
oh, this is another deprecation other than the move from internal panic in chrono: https://github.com/chronotope/chrono/pull/851

Reply:
note that `DateConfig` isn't used at all as our ci shows this api change affects no other code in our monorepo.